src/components/Common/Menu.tsx

```diff
@@ -3,13 +3,14 @@ import { DetailedHTMLProps, HTMLAttributes, ReactNode } from "react";
 
 import CareIcon from "@/CAREUI/icons/CareIcon";
 
-import { ButtonSize, ButtonVariant } from "@/components/Common/ButtonV2";
-
 import { useIsAuthorized } from "@/hooks/useIsAuthorized";
 
 import { Anyone, AuthorizedElementProps } from "@/Utils/AuthorizeFor";
 import { classNames } from "@/Utils/utils";
 
+type ButtonVariant = "primary" | "secondary" | "danger" | "warning";
+type ButtonSize = "small" | "default" | "large";
```

Comment:
let's remove these. and hardcode it directly and remove it from props too. Also mark the Dropdown Menu component as deprecated in jsdoc comments. The only usage for this is the `ExportMenu` component right now, which we can rebuild using shadcn's dropdown eventually in a later PR.

Reply:
let's also remove the `button-**` classes defined in CAREUI.css file which originally belonged to `ButtonV2`'s implementation. Do ensure it's not used anywhere else and delete it. If used else where, do remove them from there too.